src/stuf/xml.py

```diff
@@ -0,0 +1,17 @@
+"""
+XML parsing with DTD/Entities blocking.
+
+Inspired by https://github.com/mvantellingen/python-zeep/pull/1179/ as their solution
+for the deprecated defusedxml.lxml module and the defaults applied in defusedxml.lxml.
+"""
```

Comment:
It's been years since I last checked defusedxml... Is this the only vector we should worry about?